decidim-core/config/locales/en.yml

```diff
@@ -1069,7 +1069,7 @@ en:
           error: There was a problem accepting the terms and conditions.
           success: Great! You have accepted the terms and conditions.
         form:
-          agreement: I agree this terms
+          agreement: I agree with these terms
```

Comment:
OMG this has been bugging me for a while but I never spent time to open a PR, thanks!!

Reply:
The admin terms button is [still not fixed though](https://github.com/decidim/decidim/issues/6873), I opened an issue. 😁